### PR TITLE
Bluetooth: Samples: Throughput speed optimalization

### PR DESCRIPTION
**Pull request by KAGA164**

### CODE REVIEW

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -257,6 +257,7 @@ Bluetooth samples
 
   * Fixed peer throughput calculations.
     These were too low because the total transfer time incorrectly included 500ms delay without including the actual transfer.
+  * Optimize throughput speed by increasing MTU to 498 and using the maximum connection event time.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:260]:
```suggestion
  * Optimized throughput speed by increasing MTU to 498 and using the maximum connection event time.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by alwa-nordic — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 1 above)
Comment (KAGA164) [reply doc/nrf/releases/release-notes-changelog.rst]: Done

**Review by kapi-no — APPROVED** [review]

**Review by maje-emb — COMMENTED** [review]
Shouldn't we also update the `Default parameter values` table in the documentation?

**Comment by KAGA164** [comment]
> Shouldn't we also update the `Default parameter values` table in the documentation?

@maje-emb Right. I totally forgot about it. I updated the default parameters in a documentation

**Review by maje-emb — APPROVED** [review]

**Pull request merged**